x/leverage/keeper/suite_test.go

```diff
@@ -0,0 +1,183 @@
+package keeper_test
+
```

Comment:
This file contains test suite setup and helpers (some new, some relocated from `keeper_test.go`).

Now `keeper_test.go` is strictly to test the functions in `keeper.go`.